tracer/src/Datadog.Trace/Agent/MessagePack/SpanMessagePackFormatter.cs

```diff
@@ -56,6 +56,7 @@ internal class SpanMessagePackFormatter : IMessagePackFormatter<TraceChunkModel>
         private readonly byte[] _versionNameBytes = StringEncoding.UTF8.GetBytes(Trace.Tags.Version);
 
         private readonly byte[] _originNameBytes = StringEncoding.UTF8.GetBytes(Trace.Tags.Origin);
+        private readonly byte[] _lastParentIdBytes = StringEncoding.UTF8.GetBytes("_dd.parent_id");
```

Comment:
A tag called `_dd.parent_id` instead of `_dd.last_parent_id`? Isn't that going to cause confusion with `span.parent_id`?

Reply:
Yeah I agree, unfortunately nobody raised that concern to me as I was naming/re-naming the tags, so yeah it is likely too late to change it.

Hopefully it doesn't cause too much confusion, but I'll go around here and make sure that there are comments describing the different

Reply:
Thanks!